e2e/stories/dataset/output-record.e2e.ts

```diff
@@ -63,17 +74,16 @@ describe("output record fields REST API", () => {
       .select()
       .fields(fieldName)
       .subscribe((result) => {
-        const validValues = testData.map((r) => r[fieldName]);
 
         const expectedSchema = Joi
           .array()
           .items({
             id: Joi.string().uuid().required(),
-            [fieldName]: Joi.valid(validValues).required(),
```

Comment:
It was a flaky test because Joi doesn't make deep compare so that if fieldName is an object the tests fail